api/src/main/java/org/apache/iceberg/ContentFile.java

```diff
@@ -102,6 +102,18 @@
    */
   List<Long> splitOffsets();
 
+  /**
```

Comment:
@cmathiesen I think something went wrong with your rebase/merge from master, there's a lot of stuff in here now which shouldn't be.

Reply:
Yeah, trying to sort it out! :L